libindy_vdr/src/pool/cache/mod.rs

```diff
@@ -0,0 +1,65 @@
+use async_lock::RwLock;
+use async_trait::async_trait;
+use std::{fmt::Display, sync::Arc};
+
+pub mod storage;
+pub mod strategy;
+
+#[async_trait]
+pub trait CacheStrategy<K, V>: Send + Sync + 'static {
+    async fn get(&self, key: &K) -> Option<V>;
+
+    async fn remove(&mut self, key: &K) -> Option<V>;
+
+    async fn insert(&mut self, key: K, value: V, custom_exp_offset: Option<u128>) -> Option<V>;
+}
+
+pub struct Cache<K, V> {
```

Comment:
Would it make sense to require `K` to implement `Display` here as well?

Reply:
Sure thing, it's not required by rust, but now that I look it seems like a convention in the codebase